src/groundStation/system.py

```diff
@@ -3116,6 +3116,24 @@ def __init__(self):
                             }
                         }
                     },
+                    'ADCS_DOWNLOAD_FILE_LIST_TO_OBC': {
```

Comment:
Should this command also transmit a filename to save the file list as? Alternatively, add to the docs what file name is default

Reply:
There should only ever be one file list file on the OBC, so we'll just use the default "adcs_file_list.txt". Added 'what' param now so that the default name is in the command docs